containers/dev/docker-compose.yml

```diff
@@ -5,7 +5,7 @@ services:
     ports:
       - "6379:6379"
   elasticsearch5:
-    image: elasticsearch:5.6.9
+    image: elasticsearch:5.6.11
```

Comment:
The [PR](https://github.com/advthreat/tenzin/pull/1704/files#diff-7558ae5598529d6d29d7037a796d893adf3e63db30dd34d2b98504974b5be4a2R6) in tenzin upgrades it to `5.6.16-1`